src/blocks/blocks.macros.html

```diff
@@ -0,0 +1,26 @@
+{#
+  * @overview Macros for the Blocks module.
+#}
+
+{% from '../icons/icons.macros.html' import svg %}
+
+{#
+  * Renders an individual block with an icon and text beneath
+#}
+{% macro block(content, icon) %}
+<div class="block">
+  <figure class="block-image">
+    {{ svg(icon) }}
+  </figure>
+  <h4 class="block-description">{{ content }}</h4>
+</div>
+{% endmacro %}
```

Comment:
I feel like block is too generic a module name here. I also feel that the pattern inside block, commonly called media pattern, is one we'll want to address. For now, let's call this `block_item`